centreon/www/front_src/src/Header/Poller/index.tsx

```diff
@@ -0,0 +1,39 @@
+import PollerIcon from '@mui/icons-material/DeviceHub';
+
+import { MenuSkeleton } from '@centreon/ui';
+
+import ItemLayout from '../sharedUI/ItemLayout';
+
+import PollerStatusIcon from './PollerStatusIcon';
+import { PollerSubMenu } from './PollerSubMenu/PollerSubMenu';
+import { usePollerData } from './usePollerData';
+import { labelPollers } from './translatedLabels';
+
+const ServiceStatusCounter = (): JSX.Element | null => {
+  const { isLoading, data, isAllowed } = usePollerData();
+
+  if (!isAllowed) {
```

Comment:
```suggestion
  if (!isAllowed || !data) {
```

then remove the condition while returning the JSX

Reply:
ok I'll have to put the isLoading first then, 
or we will never comply to the `isLoading` condition